libcodechecker/analyze/analyzers/result_handler_plist_to_db.py

```diff
@@ -176,35 +176,13 @@ def handle_results(self):
         """
         Send the plist content to the database.
         Server API calls should be used in one connection.
-         - addBuildAction
          - addReport
          - needFileContent
          - addFileContent
-         - finishBuildAction
         """
 
         with client.get_connection() as connection:
 
-            LOG.debug('Storing original build and analyzer command '
-                      'to the database.')
-
-            _, source_file_name = os.path.split(self.analyzed_source_file)
-
-            if LoggerFactory.get_log_level() == logger.DEBUG:
-                analyzer_cmd = ' '.join(self.analyzer_cmd)
-            else:
-                analyzer_cmd = ''
-
-            build_cmd_hash = self.buildaction.original_command_hash
-            analysis_id = \
-                connection.add_build_action(self.__run_id,
-                                            build_cmd_hash,
-                                            analyzer_cmd,
-                                            self.buildaction.analyzer_type,
-                                            source_file_name)
-
-            assert self.analyzer_returncode == 0
```

Comment:
Are you sure that this piece of error checking is related to build action handling?